pkg/cmd/cli/restore/logs.go

```diff
@@ -53,3 +54,33 @@ func NewLogsCommand(f client.Factory) *cobra.Command {
 
 	return c
 }
+
+type LogsOptions struct {
+	RestoreName string
+	client      arkclient.Interface
+}
+
+func NewLogsOptions() *LogsOptions {
+	return &LogsOptions{}
+}
+
+func (l *LogsOptions) Complete(args []string) error {
+	l.RestoreName = args[0]
+	return nil
+}
+
+func (l *LogsOptions) Validate(f client.Factory) error {
+	client, err := f.Client()
+	if err != nil {
+		return err
+	}
+	l.client = client
+
+	_, err = l.client.ArkV1().Restores(f.Namespace()).Get(l.RestoreName, metav1.GetOptions{})
+	if err != nil {
```

Comment:
Can just `return err` instead of checking.

Reply:
ack